### PR TITLE
Analysis page with infinite scrolling and async export

### PR DESCRIPTION
**Pull request by anehx**
* Add new analysis page with infinite scrolling table (requires `ember-scrollable` and `ember-in-viewport`)
* Add possibility of async download with `downloadjs`

TODO:

- [x] Add tests for download
- [x] Add tests for infinite scrolling

### CODE REVIEW

File: app/analysis/controller.js [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+        /* istanbul ignore next */
+        if (!res.ok) {
+          throw new Error(res.statusText)
+        }
+
+        let file = yield res.blob()
+
+        let filename =
+          res.headers.map['content-disposition']
+            .match(/filename[^;=\n]*=((['"]).*?\2|[^;\n]*)/g)[0]
```
Comment (czosel) [inline app/analysis/controller.js]: This line might deserve a comment

Comment (anehx) [reply app/analysis/controller.js]: Agree.

File: app/analysis/template.hbs [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+            <col>
+            <col>
+            <col>
+            <col>
+            <col>
+            <col>
+            <col>
+            <col>
+            <col>
+          </colgroup>
```
Comment (czosel) [inline app/analysis/template.hbs]: For what do you need the colgroup?

Comment (anehx) [reply app/analysis/template.hbs]: `colgroup` and `col` is needed for defining column widths in a table with `table-layout: fixed;`

Comment (czosel) [reply app/analysis/template.hbs]: alright, thanks!

File: app/analysis/controller.js (hunk 1 above)
Comment (anehx) [reply app/analysis/controller.js]: Done!

**Pull request merged**